dora/minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java

```diff
@@ -149,8 +161,26 @@ protected void startProxy() throws Exception {
     }
   }
 
+  @Override
+  protected void waitForMasterServing() throws TimeoutException, InterruptedException {
```

Comment:
should we also update the waitForWorkerServing method to update the logic to verify when a worker is up? 
-> just calling the getWorkerInfo of master and see the worker size is expected or not